packages/backend/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "run-s clean build:runtime build:lib",
     "build:lib": "tsup --env.NODE_ENV production",
     "build:tests": "tsc -p tsconfig.test.json",
-    "build:runtime": "mkdir -p dist/runtime && cp -r src/runtime/* dist/runtime",
+    "build:runtime": "rsync -r --include '*/' --include '*.js' --include '*.mjs' --exclude='*' src/runtime dist",
```

Comment:
❓ shouldn't we just use `mkdir -p dist/runtime && cp -r src/runtime/*[.js|.mjs] dist/runtime` or something like that? I got confused with the `--include` and `--exclude` params.